perf/benchmark/values.yaml

```diff
@@ -33,7 +33,7 @@ server:  # server overrides
 client: # client overrides
   #tlsmode: DISABLE
   tlsmode: ISTIO_MUTUAL
-  expose: false
+  expose: true
```

Comment:
Do we need this? This creates a LoadBalancer but I don't think we use the external IP anywhere?

Reply:
Expose is only needed while testing . Do you have a different reason to expose?

Reply:
a FORTIO_CLIENT_URL is needed for runner/fortio.py (to grab the latency metrics, in order to graph them). 

But I can restore to expose=`false`.

Reply:
@mandarjog opinion on this?

Reply:
We could either go thru a gateway or use port forward to grab metrics.
But if this is too much right now, I am ok exposing it for now.